packages/fxa-settings/src/components/Settings/BentoMenu/index.tsx

```diff
@@ -34,6 +36,51 @@ export const BentoMenu = () => {
     'Mozilla products'
   );
 
+  const desktopLink = constructHrefWithUtm(
+    LINK.FX_DESKTOP,
+    'mozilla-websites',
+    'moz-account',
+    'bento',
+    'fx-desktop',
+    'permanent'
+  );
+
+  const mobileLink = constructHrefWithUtm(
+    LINK.FX_MOBILE,
+    'mozilla-websites',
+    'moz-account',
+    'bento',
+    'fx-mobile',
+    'permanent'
+  );
+
+  const monitorLink = constructHrefWithUtm(
+    LINK.MONITOR,
+    'mozilla-websites',
+    'moz-account',
+    'bento',
+    'monitor',
+    'permanent'
+  );
+
+  const relayLink = constructHrefWithUtm(
+    LINK.RELAY,
+    'mozilla-websites',
+    'moz-account',
+    'bento',
+    'relay',
+    'permanent'
+  );
+
+  const vpnLink = constructHrefWithUtm(
+    LINK.VPN,
+    'mozilla-websites',
+    'moz-account',
+    'bento',
+    'vpn',
+    'permanent'
+  );
```

Comment:
I am surprised we don't already have tests for this.

We could maybe use a single test for `constructHrefWithUtm` in `utilities.test.tsx`, and then update the render test to include the text for these products in the Bento Menu tests?

Reply:
Done!